test/unit/specs/components/Identity/getIdentity/createGetIdentity.spec.js

```diff
@@ -0,0 +1,51 @@
+import createGetIdentity from "../../../../../../src/components/Identity/getIdentity/createGetIdentity";
+
+describe("Identity::createGetIdentity", () => {
+  let sendEdgeNetworkRequest;
+  let createIdentityPayload;
+  const samplePayload = {
+    myPayload: { methodOne: () => {}, methodTwo: () => {} }
+  };
+  beforeEach(() => {
+    sendEdgeNetworkRequest = jasmine.createSpy("sendEdgeNetworkRequest");
+    createIdentityPayload = jasmine
+      .createSpy("createIdentityPayload")
+      .and.returnValue(samplePayload);
+  });
+  it("should return a function which calls sendEdgeNetworkRequest", () => {
+    const getIdentity = createGetIdentity({
+      sendEdgeNetworkRequest,
+      createIdentityPayload
+    });
+    expect(typeof getIdentity).toBe("function");
```

Comment:
The more Jasmine-y way would be `expect(getIdentity).toEqual(jasmine.any(Function));` but this works too.

Frankly, I wouldn't bother with this assertion because you're going to be calling `getIdentity` on the next line which will blow up if it's not a function anyway, plus you might feel obligated to do it on every test, but I know other people might disagree so whatever you'd like to do works with me.